From: Joachim Bauch <bauch@struktur.de>
Subject: Disable warning about missing API keys.
Description: We are building without Google API keys. A warning about disabled
 services is shown on startup - this patch disables this warning.
Index: chromium-37.0.2062.94/chrome/browser/ui/startup/google_api_keys_infobar_delegate.cc
===================================================================
--- chromium-37.0.2062.94.orig/chrome/browser/ui/startup/google_api_keys_infobar_delegate.cc	2014-08-20 21:02:04.000000000 +0200
+++ chromium-37.0.2062.94/chrome/browser/ui/startup/google_api_keys_infobar_delegate.cc	2014-11-21 14:42:15.000000000 +0100
@@ -15,11 +15,13 @@
 
 // static
 void GoogleApiKeysInfoBarDelegate::Create(InfoBarService* infobar_service) {
+#if 0
   if (google_apis::HasKeysConfigured())
     return;
 
   infobar_service->AddInfoBar(ConfirmInfoBarDelegate::CreateInfoBar(
       scoped_ptr<ConfirmInfoBarDelegate>(new GoogleApiKeysInfoBarDelegate())));
+#endif
 }
 
 GoogleApiKeysInfoBarDelegate::GoogleApiKeysInfoBarDelegate()
